Documentation/ldap-connector.md

```diff
@@ -69,25 +71,114 @@ connectors:
       emailAttr: mail
       # Maps to display name of users. No default value.
       nameAttr: name
-    # Group search configuration.
+
+    # Group search queries for groups given a user entry.
     groupSearch:
       # BaseDN to start the search from. It will translate to the query
       # "(&(objectClass=group)(member=<user uid>))".
       baseDN: cn=groups,dc=freeipa,dc=example,dc=com
       # Optional filter to apply when searching the directory.
       filter: "(objectClass=group)"
+
       # Following two fields are used to match a user to a group. It adds an additional
       # requirement to the filter that an attribute in the group must match the user's
       # attribute value.
       userAttr: uid
       groupAttr: member
+
       # Represents group name.
       nameAttr: name
 ```
 
 The LDAP connector first initializes a connection to the LDAP directory using the `bindDN` and `bindPW`. It then tries to search for the given `username` and bind as that user to verify their password.
 Searches that return multiple entries are considered ambiguous and will return an error.
 
+## Example: Mapping a schema to a search config
+
+Writing a search configuration often involves mapping an existing LDAP schema to the various options dex provides. To query an existing LDAP schema install the OpenLDAP tool `ldapsearch`. For `rpm` based distros run:
```

Comment:
@joshix new section since the old intro didn't explore what ldapsearch is and where it can be found.